Classes/Utility/AlertAction.swift

```diff
@@ -143,8 +143,18 @@ struct AlertAction {
     }
 
     static func bookmark(_ bookmark: BookmarkModel) -> UIAlertAction {
-        return UIAlertAction(title: NSLocalizedString("Add To Bookmark", comment: ""), style: .default) { _ in
-            BookmarksStore.shared.add(bookmark: bookmark)
+        let isNewBookmark = !BookmarksStore.shared.contains(bookmark: bookmark)
+        let title = isNewBookmark ? Constants.Strings.bookmark : Constants.Strings.removeBookmark
+        return UIAlertAction(
+            title: title,
+            style: isNewBookmark ? .default : .destructive
```

Comment:
Actually can we just make this `.default`? Seeing the destructive "red" next to other actions like "Close" and "Lock" is a little weird b/c this doesn't carry _nearly_ the same weight. cc @BasThomas for thoughts